practice_app/front_end/views.py

```diff
@@ -287,21 +287,36 @@ def following(request):
     return render(request, "pages/following.html", context)
 
 
-
-
-
 def follow_requests(request):
     if request.user.is_anonymous:
         return redirect("/sign_in/")
-    reqs = [
-        {'sender': 'NAME1'},
-        {'sender': 'NAME2'},
-        {'sender': 'NAME3'},
-        {'sender': 'NAME4'}
-    ]
-    context = {'page': 'Follow Requests', 'follow_requests': reqs, 'logged_in' : 1}
-    return render(request, "pages/follow_requests.html", context)
-
 
+    if request.method == "POST": 
+        button_value = request.POST.get('accept') or request.POST.get('reject')
+        if button_value:
+            action, sender_username = button_value.split('-')
```

Comment:
This line causes error since button_value.split('-') also splits ORCID id.

Reply:
Good catch, thanks! I fixed that.